packages/protocol/contracts/bridge/libs/LibBridgeRelease.sol

```diff
@@ -9,19 +9,29 @@ pragma solidity ^0.8.20;
 import { AddressResolver } from "../../common/AddressResolver.sol";
 import { EtherVault } from "../EtherVault.sol";
 import { IBridge } from "../IBridge.sol";
+import {
+    IERC165Upgradeable
+} from "@openzeppelin/contracts-upgradeable/utils/introspection/IERC165Upgradeable.sol";
 import { LibBridgeData } from "./LibBridgeData.sol";
 import { LibBridgeStatus } from "./LibBridgeStatus.sol";
 
+interface VaultContract {
+    function releaseToken(IBridge.Message calldata message) external;
+}
 /**
  * This library provides functions for releasing Ether related to message
  * execution on the Bridge.
  */
+
 library LibBridgeRelease {
     using LibBridgeData for IBridge.Message;
 
+    // All of the vaults has the same interface id
+    bytes4 public constant VAULT_INTERFACE_ID = 0x156ef222;
```

Comment:
Rename it RECALLABLE_MESSAGE_SENDER_INTERFACE_ID and define it  only once in the code (the test shall just reuse the value in the main code), to avoid unnecessary maintain of the same value in different places.